_sandboxmockedwithdrawals.md

```diff
@@ -0,0 +1,135 @@
+# Sandbox Mocked Withdrawals
+
+## Overview
+
+**Sandbox mocked withdrawals** allow teams to simulate the withdrawal flow without actually sending testnet funds. Instead, the system mocks provider calls and responses, enabling safe and comprehensive testing across environments.
```

Comment:
It might be worth making this more generic regarding the reader's profile:

```suggestion
**Sandbox mocked withdrawals** allow simulating the withdrawal flow without actually sending testnet funds. Instead, the system mocks provider calls and responses, enabling safe and comprehensive testing across environments.
```